bigbluebutton-html5/imports/api/meetings/server/handlers/meetingEnd.js

```diff
@@ -15,7 +17,7 @@ export default function handleMeetingEnd({ body }, meetingId) {
       return;
     }
     if (num) {
-      Users.find({ meetingId })
+      Users.find({ meetingId }, { fields: { meetingId: 1, userId: 1 } })
```

Comment:
Rather than finding all of the users in a meeting and then setting them to offline (double the work) why not just run the update and make the selector just the meetingId?